app/src/main/java/com/kickstarter/viewmodels/ProjectViewModel.kt

```diff
@@ -1069,6 +1082,9 @@ interface ProjectViewModel {
         @NonNull
         override fun startCommentsActivity(): Observable<Pair<Project, ProjectData>> = this.startCommentsActivity
 
+        @NonNull
+        override fun startCommentsThreadedActivity(): Observable<Pair<Project, ProjectData>> = this.startCommentsThreadedActivity
```

Comment:
Nit: I think to keep parity with iOS, we are eventually going to name the comments activity `RootCommentsActivity`, so you may want to change the name of this method from threaded to root

Reply:
Done!